src/main/java/core/basesyntax/services/FruitsServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.services;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.transactions.FruitsTransaction;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitsServiceImpl implements FruitsService {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private final FruitsDao fruitsDao;
```

Comment:
this class is doing two very different things - processing transactions and generating report, consider breaking it up into two to follow SRP